mix.exs

```diff
@@ -11,21 +11,20 @@ defmodule Silicon.MixProject do
     ]
   end
 
-  # Run "mix help compile.app" to learn about applications.
   def application do
     [
       extra_applications: [:logger]
     ]
   end
 
-  # Run "mix help deps" to learn about dependencies.
   defp deps do
     [
       {:rustler, "~> 0.30.0", runtime: false},
-      {:ex_doc, "~> 0.31", only: :dev, runtime: false},
+
+      # Developer tooling
       {:type_check, "~> 0.13.3"},
```

Comment:
`type_check` shouldn't be under dev tooling

Reply:
Isn't it though? It doesn't provide any end-user features as far as I am aware?

Reply:
Currently, it's only used for runtime validation since rustler's suck

Reply:
Ah so it is used at runtime and not only during development like `mix dialyzer`? If so then I'll move it as recommended.